ArrayOps.java

```diff
@@ -1,26 +1,173 @@
 public class ArrayOps {
     public static void main(String[] args) {
-        
+        //int[] array={1,-2,3,-4,5};
+        //System.out.println(findMissingInt(array));
+        //System.out.println(secondMaxValue(array));
+        int [] array2={1,2,3,-4,5};
+        int [] array1={1,3,-4,5};
+        System.out.println(containsTheSameElements(array1,array2));
+        //int [] array={1,1,1,2,2};
+        //System.out.println(isSorted(array));
     }
     
+    //returns the missing integer in the array, between 0 and the array length
     public static int findMissingInt (int [] array) {
         // Write your code here:
-        return -1;
+        int sumArray=0;
+        int supposedSumArray=0;
+        int missing=0;
+
+
+        for(int i=1;i<=array.length;i++)
+        {
+            supposedSumArray+=i;
+        }
+
+        for(int j=0;j<array.length;j++)
+        {
+            sumArray+=array[j];
+        }
+
+        missing=supposedSumArray-sumArray;
+        return missing;
     }
 
+
     public static int secondMaxValue(int [] array) {
         // Write your code here:
-        return 0;
+        int max=array[0];
+        int tempI=0;
+        for(int i=1;i<array.length;i++)
+        {
+            if(array[i]>max)
+            {
+                max=array[i];
+                tempI=i;
+            }
+        }
+
+        int max1=array[0];
+        for(int j=1;j<array.length;j++)
+        {
+            
+            if(array[j]>max1 && tempI!=j) //&& array[j]!=max
+            {
+                max1=array[j];
+            }
+            else
+            {
+                if(array[j]==max && j!=tempI)
+                {
+                    max1=array[j];
+                    j=array.length; //break;
+                }
+            }
```

Comment:
No need to nest that if
```suggestion
            else if(array[j]==max && j!=tempI)
            {
                 max1=array[j];
                 j=array.length; //break;
            }
```